### ISSUE TITLE
Question regarding finals submission

### ISSUE THREAD

**Issue by YEETTTTTTT** [nus-cs2030/2223-s1#346]
Hi, do we only submit our code to codecrunch? or do we need to submit it in luminus as well? Asking because there is a final submission folder in the finals assessment (mock) folder

**Comment by cyiting** [comment]
I believe we have to submit the zip file to luminus as well.

**Comment by Grauda123** [comment]
Remembered prof mentioned that the main place to submit our code is on Codecrunch. The final submission folder is a backup

**Comment by Nicklelodeon** [comment]
I think codecrunch submission is the main one, while the luminus submission is the backup one but can submit both to be safe!

**Comment by ZYN2122** [comment]
Yep yep! And remember to submit before the time is up for the exam also

**Comment by mohddanesh** [comment]
you ONLY sumbit to luminus if you submit the wrong file on codecrunch. I got confirm with prof about this before. so you shouldn't submit to luminus if you submit the right file on codecrunch

**Comment by YEETTTTTTT** [comment]
Thank you for the answers, btw just to confirm we submit the answers back as a zip file?

**Comment by mohddanesh** [comment]
yup we submit zip file

**Comment by nguyenhpnguyen** [comment]
wait I thought we should submit the answer files (already unzipped) on codecrunch, only when we submit to luminus we will submit a zip file?
